src/app/beer_garden/config.py

```diff
@@ -757,6 +757,15 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
                     "previous_names": ["info_request_ttl"],
                     "alt_env_names": ["INFO_REQUEST_TTL"],
                 },
+                "in_progress": {
+                    "type": "int",
+                    "default": 15,
```

Comment:
Another thing I missed last time: this should default to `-1`.  That way the default behavior is what people were already used to (we don't cancel outstanding requests) and they have to opt into this new behavior by explicitly setting this.